launchDarkly/server/initNodeSdk.ts

```diff
@@ -0,0 +1,18 @@
+export const initNodeSdk = async () => {
+  if (process.env.NEXT_RUNTIME === 'nodejs') {
+    const sdk = await import('@launchdarkly/node-server-sdk');
+    const nodeClient = sdk.init(process.env.LAUNCHDARKLY_SDK_KEY ?? '');
+
+    // HACK: save this globally so we can access it from the app
+    global.nodeClient = nodeClient;
```

Comment:
callout: If you have a better suggestion other than global lmk. I tried and this is the only thing that works.

Reply:
I think this is the best way - I was trying a few different things, but the ergonomic of yours is really good - and prevents them having to initiate a server client over and over again. 

How would you feel about calling it something different? Like LDserverClient or something like that? I just want to make the ergonomic easy for a customer to use - and nodeClient sounds much more like part of the core node process.

I'm very open to back and forth on this?

Reply:
Ok I'm renaming to `ldNodeClient` as discussed on slack.